system-variables.md

```diff
@@ -437,6 +437,47 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - This variable is used to change the default priority for statements executed on a TiDB server. A use case is to ensure that a particular user that is performing OLAP queries receives lower priority than users performing OLTP queries.
 - You can set the value of this variable to `NO_PRIORITY`, `LOW_PRIORITY`, `DELAYED` or `HIGH_PRIORITY`.
 
+### tidb_gc_concurrency
+
+- Scope: GLOBAL
+- Default: -1 (Automatic)
+- Specifies the number of concurrenct garbage collection threads to be used. A value of `-1` means that TiDB will automatically decide the number of garbage collection threads to use.
+- GC concurrency works in the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step.
```

Comment:
Actually you can just say "Specifies the number of threads in the [Resolve Locks] step of GC.